src/normalize/entities.ts

```diff
@@ -212,6 +212,18 @@ export const CalendarItemCreateModifyRequest = new Entity({
 	invId: 'inviteId'
 });
 
+export const InstanceDate = new Entity({
+	d: 'date'
+});
+
+export const CalendarItemDeleteRequest = new Entity({
+	inst: ['instanceDate', InstanceDate],
+	id: 'inviteId',
+	comp: 'componentNumber',
```

Comment:
Can we call this `componentNum` since there are already a few of those in this file?

Reply:
Resolved above review comment.